django-backend/fecfiler/openfec/test_efo_mock_data/committee_accounts.json

```diff
@@ -2,46 +2,487 @@
     {
         "committee_id": "C00100214",
         "name": "TEST_COMMITTEE_NAME3",
-        "committee_type_full": "TEST_COMMITTEE_TYPE3"
+        "committee_type_full": "TEST_COMMITTEE_TYPE3",
+        "street_1": "TEST_SA3",
+        "city": "TEST_CITY3",
+        "state": "AL",
+        "zip": "12345",
+        "email": "fakeemail@test.com",
```

Comment:
any reason we're using `fakeemail@test.com` instead of our actual emails?  we'll need to overwrite the email every time we submit

Reply:
I believe @Elaine-Krauss-TCG suggested this so that we aren't storing our emails in a public repo.  I'm fine either way though.

Reply:
I think it a good general policy not to put real emails in the public repo.